VideoWeb/VideoWeb.AcceptanceTests/Features/WaitingRoom.feature

```diff
@@ -18,7 +18,6 @@ Scenario: Participant waiting room
 Scenario: Judge waiting room
 	Given the Judge user has progressed to the Waiting Room page
 	Then the user is on the Waiting Room page
-  And the judge dismisses the change camera popup
 	And a phone number for help is provided
```

Comment:
Can you remove this from the test is Accessibility.feature as well please?

Reply:
Removed